Client/Frontend/Reader/Reader.html

```diff
@@ -46,6 +40,13 @@ <h1 id="reader-title"></h1>
     <li><a id="toggle-button" class="button toggle-button" href="#"></a></li>
   </ul>
 
+  <script>
+      document.getElementById("reader-page-title").textContent = "%READER-TITLE%";
+      ["title", "credits", "content", "message"].forEach((name) => {
+        document.getElementById(`reader-${name}`).textContent = `%READER-${name.toUpperCase()}`;
```

Comment:
Just realized some pages render as HTML. Everything else can be `textContent` except for `reader-content` which can render HTML via `innerHTML`.

Still reviewing to see if any other websites break it.